waiter/src/waiter/token.clj

```diff
@@ -264,10 +276,7 @@
               owner->index-entries (pc/map-vals
                                      (fn [tokens]
                                        (pc/map-from-keys
-                                         (fn [token]
-                                           (let [{:strs [deleted last-update-time maintenance] :as token-data} (kv/fetch kv-store token)
-                                                 token-hash (sd/token-data->token-hash token-data)]
-                                             (make-index-entry token-hash deleted last-update-time maintenance)))
+                                         (partial get-token-index kv-store)
```

Comment:
Please uses inline function instead of `partial`. We have found performance issues with use of `partial` in the past.
```suggestion
                                         #(get-token-index kv-store % :include #{})
```